### PR TITLE
First cut at updating docs to reflect Dogecoin

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -137,35 +137,21 @@ rm SHA256SUMS
...
-    instructions: https://github.com/bitcoin/bitcoin.org#release-notes
-
-  - After the pull request is merged, the website will automatically show the newest version within 15 minutes, as well
-    as update the OS download links. Ping @saivann/@harding (saivann/harding on Freenode) in case anything goes wrong
+- Update dogecoin.com version - Langerhans to do
 
 - Announce the release:
 
-  - Release sticky on bitcointalk: https://bitcointalk.org/index.php?board=1.0
+  - Release sticky on dogecointalk: https://bitcointalk.org/index.php?board=1.0
```
Comment (langerhans) [inline doc/release-process.md]: Link needs fix

File: doc/release-process.md [hunk 2]
```diff
@@ -137,35 +137,21 @@ rm SHA256SUMS
...
+  - Dogecoin-development mailing list
 
-  - Update title of #bitcoin on Freenode IRC
+  - Update title of #dogecoin on Freenode IRC
 
-  - Optionally reddit /r/Bitcoin, ... but this will usually sort out itself
+  - Announce on reddit /r/dogecoin, /r/dogecoindev
 
-- Notify BlueMatt so that he can start building [https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin](the PPAs)
+- Release sticky on discuss dogecoin: https://discuss.dogecoin.com/categories/announcements
```
Comment (langerhans) [inline doc/release-process.md]: Doesn't exist anymore

File: doc/init.md [hunk 3]
```diff
@@ -1,61 +1,61 @@
...
-    contrib/init/bitcoind.service:    systemd service unit configuration
-    contrib/init/bitcoind.openrc:     OpenRC compatible SysV style init script
-    contrib/init/bitcoind.openrcconf: OpenRC conf.d file
-    contrib/init/bitcoind.conf:       Upstart service configuration file
-    contrib/init/bitcoind.init:       CentOS compatible SysV style init script
+    contrib/init/dogecoind.service:    systemd service unit configuration
+    contrib/init/dogecoind.openrc:     OpenRC compatible SysV style init script
+    contrib/init/dogecoind.openrcconf: OpenRC conf.d file
+    contrib/init/dogecoind.conf:       Upstart service configuration file
+    contrib/init/dogecoind.init:       CentOS compatible SysV style init script
```
Comment (langerhans) [inline doc/init.md]: We didn't rename them, or edit them at all.

Comment (rnicoll) [inline doc/init.md]: Oops, tackled in #1250

**Pull request merged**